src/simulation/dynamics/constraintEffector/constraintDynamicEffector.h

```diff
@@ -82,7 +87,14 @@ class ConstraintDynamicEffector: public SysModel, public DynamicEffector {
     /** getter for `c_a` gain */
     double getC_a() const {return this->c_a;};
 
+public:
+
+    Message<ConstDynEffectorMsgPayload> constraintElements;
+    ReadFunctor<DeviceStatusMsgPayload> effectorStatusInMsg;
+    uint64_t effectorStatus=1;
```

Comment:
can we make `effectorStatus` a `bool` instead of `uint64_t`?  This would be

`bool effectorStatus=true;`

Reply:
The DeviceStatus message type supports an enum type having 0 for "Off" and 1 for "On". This was why I decided to go with a uint64_t datatype for this. If we have to use a boolean data type we might have to tweak the DeviceStatus input message definition.